spec.bs

```diff
@@ -2444,6 +2471,17 @@ dictionary BiddingBrowserSignals {
   object wasmHelper;
   unsigned long dataVersion;
 };
+
+dictionary ScoringBrowserSignals {
+  required DOMString topWindowHostname;
+  required USVString interestGroupOwner;
+  required USVString renderURL;
+  required unsigned long biddingDurationMsec;
+  required DOMString bidCurrency;
+
+  unsigned long dataVersion;
+  sequence<USVString> adComponents;

```

Comment:
Is there a difference between an `adComponents` that's an empty list and one that's undefined? If so, you should describe it in this dictionary's `domintro`. If not, I'd make this field `required` and use the empty list where you're currently leaving it un-set.

Reply:
no difference. Just that our implementation set this field as optional. Changed to required.